### ISSUE TITLE
Ability to import requests from Postman

### ISSUE THREAD

**Issue by mmjsmohit** [foss42/apidash#52]
**Tell us about the task you want to perform and are unable to do so because the feature is not available**
Currently, there is no option to import JSON Collection exported from Postman into API Dash. If this feature is added, it would make the process of switching from Postman very swift.

**Describe the solution/feature you'd like us to add**
It would be nice if there was an interface where user can add collection exported from Postman into API Dash.

**Any other feedback you would like to provide regarding the site**
No
![image](https://github.com/foss42/api-dash/assets/65406090/6a035c8f-2555-49da-bc49-406ef81ea793)

Related Documentation:
[Postman essentials: exploring the collection format](https://blog.postman.com/postman-essentials-exploring-the-collection-format/)
[Travelogue of Postman Collection Format v2](https://blog.postman.com/travelogue-of-postman-collection-format-v2/)
[Using collections](https://learning.postman.com/docs/collections/using-collections/)

**Comment by mmjsmohit** [comment]
I would like to work on this issue!

**Comment by mmjsmohit** [comment]
Hi! @animator 
Since the PR linked to #24 is merged and released in the latest version, can I work on this issue now?

**Comment by animator** [comment]
There is still some stuff left before we can start this. Will update this thread.

**Comment by ashitaprasad** [comment]
https://github.com/foss42/apidash/issues/461

**Comment by ahmedtohamy1** [comment]
@ashitaprasad so what's the progress on this request

**Comment by ashitaprasad** [comment]
Import from Postman Collection is now live in main branch.

**Comment by hafijulali** [comment]
I tried importing a collection from postman, but the folders were not created by apidash, however it did import individual requests.

Built app locally from main branch on macos M2.

Please let me know if this is a known issue.

**Comment by ashitaprasad** [comment]
@hafijulali Please raise  a new issue for this feature. Thanks.

**Comment by animator** [comment]
@hafijulali Can you please raise a new issue for this feature.
Thank you.

**Comment by hafijulali** [comment]
@animator here is the issue id : #879 please take a look.
